hack/install-deps.sh

```diff
@@ -42,7 +42,7 @@ go get -d ${RUNC_PKG}/...
 cd ${GOPATH}/src/${RUNC_PKG}
 git fetch --all
 git checkout ${RUNC_VERSION}
-make
+make BUILDTAGS='seccomp apparmor'
```

Comment:
We should make apparmor optional via an environment variable. Let's not build it by default, and only enable it in our travis.

I don't think people using selinux need apparmor built in. :)